postgres/src/main/scala/postgres/clients/BulkSenderPostgres.scala

```diff
@@ -94,7 +95,7 @@ class BulkSenderPostgres(
           {
             if (filterVals.nonEmpty) {
               val filteredRecords = recordsForPartition.filter { rec =>
-                val extractedValueOption = extractStringElementFromJson(PostgresFilterTypes.APP_ID, rec.json)
+                val extractedValueOption = extractStringElementFromJson(PostgresFilterTypes.APP_ID, parse(rec.json.toString()).asInstanceOf[JObject])
```

Comment:
This goes with my comment in JsonRecord.scala class, where we have changed json: JValue from json: JObject.
so it was JObject this transformation need not be done.